test/dlc_tests/sanity/test_mxnet_blasmkl.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from invoke.context import Context
+from packaging.version import Version
+
+from test.test_utils import (
+    get_container_name,
+    get_framework_and_version_from_tag,
+    get_processor_from_image_uri,
+    run_cmd_on_container,
+    start_container,
+    stop_and_remove_container,
+)
+
+@pytest.mark.usefixtures("sagemaker")
+@pytest.mark.model("N/A")
+@pytest.mark.processor("cpu")
+@pytest.mark.integration("mxnet_blasmkl_sanity")
+def test_mxnet_blasmkl_sanity(mxnet_inference):
```

Comment:
Thanks for adding the test.

nit: do we want to call it `test_mxnet_blas_mkl_sanity` instead of `test_mxnet_blasmkl_sanity`?

Reply:
Resolved.